oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/query/ElasticIndex.java

```diff
@@ -46,8 +46,8 @@
     // no concept of rewound in ES (even if it might be doing it internally, we can't do much about it
     private static final IteratorRewoundStateProvider REWOUND_STATE_PROVIDER_NOOP = () -> 0;
 
-    // # OAK-9419 
-    private static final double MIN_COST = 2.2;
+    // #OAK-9457 Further changed from 2.2 to 2.1 so that elastic and lucene are at par in terms of min cost
+    private static final double MIN_COST = 2.1;
```

Comment:
this constant is duplicated in `ElasticIndex` and `LucenePropertyIndex`. What about having it only in `FulltextIndex`? Even the method `getMinimumCost` can be specified in `FulltextIndex` only and overridden in subclasses when needed.